### PR TITLE
Minor code refactoring

### PR DESCRIPTION
**Pull request by rshanmugam-enquizit**
### Notes

This PR includes minor refactoring that includes the changes to the variable name and addition of username and password.

```
➜  NEDSS-DataIngestion-CLI git:(minor-code-refactoring) ✗ ./build/native/nativeCompile/nbs-di-cli token --username --password
Enter value for --username (Username to connect to DI service): diteamadmin
Enter value for --password (Password to connect to DI service):
eyJhbGciOiJSUzI1NiJ9.eyJpc3MiOiJzZWxmIiwic3ViIjoiZGl0ZWFtYWRtaW4iLCJleHAiOjE2OTg2ODY2ODgsImlhdCI6MTY5ODY4MzA4OCwic2NvcGUiOiJBRE1JTiJ9.WDta8CRIIuk_hTYdv9EmgKECmFG52dQApbyqBQ0BMeX7AqfA_uV765eMxU4HSynIp1I4mTJ8QncbU_T1qlcxSu2L_kuTWHO8IrmCbAI_Wp187OJDR4S4QySBu4imloCU2qd-VwVM7w9xkZupHq9L5LWtJsSSf3Xx99tHYZfixrr3uuBcF6urXnRwV9ztYMGL6Qn9msa4JYo9QJKQ7zhVITgGLd-jl1ca-WhX3t3BTatSeq6MDlamjS7WMukqDDQFbpjGVTj9JWcWDJB50eyz3_Xd3EYD55g1ICww2IcO-Vs85r6ZVLCg4rKIx0YCQ556_ZVsSN5CuGLw0QhTbEwNAw
```

```
➜  NEDSS-DataIngestion-CLI git:(minor-code-refactoring) ✗ ./build/native/nativeCompile/nbs-di-cli injecthl7 --hl7-file --username --password
Enter value for --hl7-file (HL7 file name with fully qualified path): /Users/RagulShanmugam/Documents/hl7-input-file.hl7
Enter value for --username (Username to connect to DI service): diteamadmin
Enter value for --password (Password to connect to DI service):
8C2C6CDA-A25E-48D8-AF64-0D5671ECB028
```

```
➜  NEDSS-DataIngestion-CLI git:(minor-code-refactoring) ./build/native/nativeCompile/nbs-di-cli status --report-id --username --password
Enter value for --report-id (UUID provided by Data Ingestion Service during report ingestion): 9E29682F-28AE-4051-9A60-066A1B04C1FD
Enter value for --username (Username to connect to DI service): diteamadmin
Enter value for --password (Password to connect to DI service):
{"id":"9E29682F-28AE-4051-9A60-066A1B04C1FD","status":"Failure"}
```

```
➜  NEDSS-DataIngestion-CLI git:(minor-code-refactoring) ✗ ./build/native/nativeCompile/nbs-di-cli injecthl7 --username=clicli --password=cliclicli --hl7-file=/Users/RagulShanmugam/Documents/HL7_input
HL7 file not found at the given location.
```

```
➜  NEDSS-DataIngestion-CLI git:(minor-code-refactoring) ✗ ./build/native/nativeCompile/nbs-di-cli register --client-username --client-secret --admin-user --admin-password
Enter value for --client-username (Username provided by the client): cli
Enter value for --client-secret (Secret provided by the client):
Enter value for --admin-user (Admin Username to connect to DI service): diteamadmin
Enter value for --admin-password (Admin Password to connect to DI service):
The username must be atleast six characters in length.
```

### JIRA
- https://cdc-nbs.atlassian.net/browse/CNDIT-761

### Testing
- Added unit tests with 80% coverage
- Tested it in local and added the output to this PR

Screenshot 2023-11-10 at 9 31 11 AM

### CODE REVIEW

File: src/main/java/gov/cdc/dataingestion/util/AuthUtil.java [hunk 1]
```diff
@@ -19,7 +19,13 @@ public class AuthUtil {
 
     public String getResponseFromDIService(AuthModel authModel, String name) {
         try {
-            UsernamePasswordCredentials credentials = new UsernamePasswordCredentials(authModel.getAdminUser(), new String(authModel.getAdminPassword()));
+            UsernamePasswordCredentials credentials;
+            if(name.equals("register")) {
```
Comment (alog-enquizit) [inline src/main/java/gov/cdc/dataingestion/util/AuthUtil.java:23]: Is the register response only applicable to the admin user alone?

Comment (rshanmugam-enquizit) [reply src/main/java/gov/cdc/dataingestion/util/AuthUtil.java:23]: Yes, registration functionality can only be used by the administrator.

**Review by alog-enquizit — APPROVED** [review]
lgtm 👍

**Review by ssathiah — APPROVED** [review]
Looks good.

**Pull request merged**